bin/seed_test_users.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+numberExpression='^[0-9]+$'
+databasePath=var/user-lifecycle.sqlite
+
+if ! [[ $1 =~ $numberExpression ]] ; then
+   echo "Usage: The first argument must be a number. Indicating the number of identities you'd like to seed to the last_login table" >&2; exit 1
+fi
+
+if ! [[ $2 =~ $numberExpression ]] ; then
+   echo "Usage: The second argument must be a number. Indicating the number of days since last login for all users" >&2; exit 1
+fi
+
+if ! [ -x "$(command -v sqlite3)" ]; then
+    echo "The SQLite binary is not available"
+fi
+if ! [ -x "$(command -v rig)" ]; then
+    echo "The rig binary is not available, Rig is used to create a random username"
```

Comment:
Same here: You might want to exit here. You could also use already available commands to generate something random, like:
` head /dev/urandom |  tr -dc A-Za-z0-9 | head -c 13 ; echo '`

Reply:
Leaving this as-is for now. If ever RIG starts working against us (as in it becomes difficult to install on a test/dev rig) I will get rid of it then.